### DISCUSSION TITLE
PAG File (Panel)中进行勾选操作，AE就会崩溃

### DISCUSSION THREAD

**Discussion by dukai6604** [Tencent/libpag#2283] [Q&A] [answered]
<img width="531" alt="截屏2024-05-07 11 21 27" src="https://github.com/Tencent/libpag/assets/62284392/173097c9-9796-49db-8945-2d5f783aa7c2">
<img width="486" alt="截屏2024-05-07 11 23 02" src="https://github.com/Tencent/libpag/assets/62284392/b263150e-cf29-4b1e-917f-ee6506786d0f">

如题，PAG File (Panel)中进行勾选操作，会导致AE崩溃。尝过AE2023、2024，PAG最新版本、4.0.5、4.1.1，换了几台mac电脑都会出现此问题。身边其他人也同遇到此问题，windows下也如此。

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在请描述下复现路径和场景，包括系统版本、AE版本、PAGViewer版本等，因为该问题在我本地没有复现到，需要了解下复现路径，如果方便请提供下问题对应的aep、pag文件，用于复现该问题

**Comment by jinwuwu001** [comment, marked as answer]
该问题的原因是AE在向macOS临时文件夹写入文件时没有权限导致的，可通过如下路径给AE添加权限
设置-->隐私与安全性-->完全磁盘访问权限-->给AE2025添加权限，如下图所示给AE添加权限
<img width="490" alt="给AE添加权限" src="https://github.com/user-attachments/assets/ae3ef64e-9822-43e7-97c1-5859f4e99ab1" />
通过上述操作后可临时规避权限问题导致的写入失败，可使用官网最新版PAGViewer安装AE插件后进行导出。
https://pag.io/docs/pag-download.html
AE导出插件后续版本中会修复该问题。
